dev-docs/bidders/gumgum.md

```diff
@@ -37,6 +37,16 @@ Client side and server side parameters differ slightly. For Server side (Prebid
 ### Client Side Bid Params
 
 {: .table .table-bordered .table-striped }
+<<<<<<< HEAD
```

Comment:
Looks like there are some issues here, with the table duplicated. Maybe there was a conflict? Can you take a look and correct this?